commit b7bdbce799d20ea3344998888da1bea1872d8205 (refs/changes/97/4597/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2018-07-27T13:50:40-07:00 (1 year, 2 months ago)
    
    TRITON-453 docker build failing in nightly test run due to vmapi BadDigest error

diff --git a/package.json b/package.json
index 77c4a3a..af448df 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.6.0",
+  "version": "9.6.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -31,7 +31,7 @@
     "triton-tags": "^1.1.4",
     "vasync": "2.2.0",
     "verror": "1.10.0",
-    "wf-client": "0.2.1"
+    "wf-client": "0.3.1"
   },
   "sdcDependencies": {
     "vm-agent": ">=1.5.0"
